dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sfdr/categories/environmental/subcategories/SfdrEnvironmentalGreenhouseGasEmissions.kt

```diff
@@ -1,41 +1,70 @@
 package org.dataland.datalandbackend.model.sfdr.categories.environmental.subcategories
 
+import jakarta.validation.Valid
 import org.dataland.datalandbackend.model.datapoints.CurrencyDataPoint
 import org.dataland.datalandbackend.model.datapoints.ExtendedDataPoint
 import org.dataland.datalandbackend.model.enums.commons.YesNo
+import org.dataland.datalandbackend.validator.DataPointMinimumValue
 import java.math.BigDecimal
 
 /**
  * --- API model ---
- * Fields of the subcategory "Greenhouse gas emissions " belonging to the category "Environmental" of the sfdr
- * framework.
- */
+ * Fields of the subcategory "Greenhouse gas emissions "
+ * belonging to the category "Environmental" of the sfdr framework.
+*/
 data class SfdrEnvironmentalGreenhouseGasEmissions(
+    @field:Valid
+    @field:DataPointMinimumValue(minimumValue = -10000000000)
```

Comment:
Why those minimum values?

Reply:
i removed the generation of these min values in the dev tools now

Reply:
Gone now, so i will close this here